.vscode/tasks.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
If you can remove this file from the PR it would be great. 
We haven't decided on config files for editors yet.

Reply:
How do you currently debugging the tests?

Reply:
The same way, we just haven't decided if we want to put editor config inti the code. Once we have our answer there might be a thread were ppl get to vote on vscode idea etc.